### DISCUSSION TITLE
Moosync App

### DISCUSSION THREAD

**Discussion by ohxxm** [catppuccin/catppuccin#1840] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes, finished mocha version.
UPDATE: Apparently, you cannot import/export themes in this app, so it would require the end-user to manually add the colors in, I have opened an issue for this feature to be implemented. You can check the issue here: https://github.com/Moosync/Moosync/issues/198

**Comment by ohxxm** [comment]
So an update has been pushed out where you can import/export themes. I have finished the port for Catppuccin. If this is moved to the catppuccin organization, i shall add the license and readme. Here's the repo: https://github.com/Spaxly/catppuccin-moosync

**Comment by sgoudham** [comment]
Closed as discussion has moved to #2115
